services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -105,7 +105,10 @@ class HankeServiceITests {
 
         assertThat(returnedHanke.version).isZero
         assertThat(returnedHanke.createdAt).isNotNull
-        assertThat(returnedHanke.createdAt!!.toEpochSecond() - currentDatetime.toEpochSecond()).isBetween(-600, 600) // +/-10 minutes
+        assertThat(returnedHanke.createdAt!!.toEpochSecond() - currentDatetime.toEpochSecond()).isBetween(
```

Comment:
Uh ei.. jos rivi on pakko pilkkoa, niin sitten .isBetween(-600, 600) seuraavalle riville. Ei todellakaan näin, että nopeasti lukien näyttäisi siltä kuin -600 ja 600 olisi parametreja assertThat(... -funktiolle

Reply:
Korjattu